extra_data/export.py

```diff
@@ -242,7 +254,24 @@ def serve_files(path, port, source_glob='*', key_glob='*', **kwargs):
     streamer.start()
     for tid, train_data in data.trains():
         if train_data:
-            streamer.feed(train_data)
+            if data_like == 'file':
+                streamer.feed(train_data)
+            elif data_like == 'online':
+                stacked = stack_detector_data(train_data, 'image.data')
```

Comment:
I think this will fail if you have any other sources selected. That's not necessarily a problem, but we should document it.

Or we could make it so that the option to stack detector modules automatically selects only the detector source from the files. But that would need to be documented too.

Reply:
My choice of source name and method corresponds to the current use-case: AGIPD appender data from the SPB online calibration pipeline. It is desirable to extend and thus generalize this, but maybe we can think of a short-term solution which is, as you say, documented.  
The alternative would be that we offer a tentative version to be installed from the branch. Which is probably better than a "unsupervised" (not version controlled etc.) script like that currently used for testing.

Reply:
Do you think it would be too much effort to generalize it?
We could try to use the Detector classes form `extra_data.components`, would that work?